.github/workflows/actions.yml

```diff
@@ -31,9 +31,10 @@ jobs:
           src: .
       - name: Install dependencies
         run: |
-          pip install -e .[dev] \
-            -e benchadapt/python \
-            -e benchrun/python
+          pip install -e benchadapt/python --no-deps && \
```

Comment:
This is janky but necessary for now because it tries to install benchclients from main, which it won't be on till this PR merges. We can revert this right after.

Reply:
ah makes sense. We don't have to revert it yet; I'll try to fix it the "right" way when adding integration tests.

Reply:
Yeah, the reason we'd need to revert is if benchadapt adds other deps for some reason. It's deliberately lightweight, so I don't foresee it, but it's possible.